src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -197,6 +205,11 @@ def create(input_dataset, target, feature=None, validation_set='auto',
         raise TypeError("Unrecognized type for 'validation_set'."
             + validation_set_corrective_string)
 
+    dataset = _drop_missing_values(dataset, feature, is_train =True)
+
+    if validation_set is not None:
```

Comment:
Nit: `validation_dataset` is always an (possibly empty) `SFrame` at this point, right? Could we just unconditionally do line 211?